dotCMS/src/main/java/com/dotcms/graphql/datafetcher/SiteFieldDataFetcher.java

```diff
@@ -13,22 +15,24 @@
 import graphql.schema.DataFetcher;
 import graphql.schema.DataFetchingEnvironment;
 
-public class SiteFieldDataFetcher implements DataFetcher<Map<String, Object>> {
+public class SiteFieldDataFetcher implements DataFetcher<Contentlet> {
     @Override
-    public Map<String, Object> get(final DataFetchingEnvironment environment) throws Exception {
+    public Contentlet get(final DataFetchingEnvironment environment) throws Exception {
         try {
             final User user = ((DotGraphQLContext) environment.getContext()).getUser();
             final Contentlet contentlet = environment.getSource();
-            final Map<String, Object> siteMap = new HashMap<>();
```

Comment:
get a reference of the inner map here and over it. instead of having to do host.getMap at all times

Reply:
Done